src/index.ts

```diff
@@ -28,6 +31,9 @@ export {
   IRoleDefinitionV2,
   IRoleDefinitionText,
   PreconditionType,
+  IFieldDefinition,
+  IIssuerDefinition,
+  IRevokerDefinition,
```

Comment:
Why not to reexport them?

Reply:
What do you mean? I just fallowed previous convention with importing and then exporting.

Reply:
Yes, I know it is not related to PR, but taking chance this file can be simplified. It is up to you though

Reply:
I will create a task for that :+1:

Reply:
ICL-250